src/main/java/it/aboutbits/springboot/emailservice/lib/application/EmailServiceMigrator.java

```diff
@@ -0,0 +1,70 @@
+package it.aboutbits.springboot.emailservice.lib.application;
+
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.jdbc.core.JdbcTemplate;
+
+@Slf4j
+public class EmailServiceMigrator {
+    private final JdbcTemplate jdbcTemplate;
+
+    public EmailServiceMigrator(JdbcTemplate jdbcTemplate) {
+        this.jdbcTemplate = jdbcTemplate;
+    }
+
+    void migrate() {
```

Comment:
What happens if the migration fails for some reason?
It would be nice that the application fails to start in that case, similar to what happens with Liquibase. Is this already the case?

Reply:
We agreed that this will be tested in Aichner and in case it doesn't work we change it.